cli/CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 12.13.1
+
+_Released 05/31/2023 (PENDING)_
+
+**Bugfixes:**
+
+- Attempts to fix an intermittent issue where chrome crashes when connecting to a new spec. Addresses [#24650](https://github.com/cypress-io/cypress/issues/24650).
```

Comment:
Do we know if this actually addresses the issue that you and others are running into it, or is it difficult to tell because it seems to happen sparingly?

Reply:
I'm not sure at this point because i can't reproduce locally. It only seems to happen in our jenkins build pipeline. So it's more of a hope than a definite fix.